tripServer/requestHandlers.js

```diff
@@ -0,0 +1,51 @@
+const request = require('axios');
+
+
+const APIKEY = process.env.APIKEY;
+
+const googleMapsURL = 'https://maps.googleapis.com/maps/api/directions/json?origin=';
+
+const requestHandler = (req, resp) => {
+  // define origin and destination from request parameters
+  // hard coded locations for now
+  const origin = '37.783669,-122.40895';
+  const destination = '37.781256,-122.405955';
+  const directionsRequestURL = googleMapsURL + origin + '&destination=' + destination + '&mode=walking&key=' + APIKEY;
```

Comment:
I know we already discussed this but just as a reminder, I think you should create an object with origin and destination then append it to the URL as a query string.

I'm using the 'qs' module to create a query string from an object, but we can discuss how we want to do this across all components of the app so we're all doing the same thing.